quizler/utils.py

```diff
@@ -53,6 +53,40 @@ def print_common_terms(common_terms: List[Tuple[str, str, Set[str]]]):
                 print('    {}'.format(term))
 
 
+def delete_term(set_id, term_id, access_token):
+    """Delete the given term."""
+    api_call('delete', 'sets/{}/terms/{}'.format(set_id, term_id), access_token=access_token)
+
+
+def add_term(set_id, term: Term, access_token):
+    """Add the given term to the given set."""
+    api_call('post', 'sets/{}/terms'.format(set_id), term.to_dict(), access_token=access_token)
+
+
+def reset_term_stats(set_id, term_id, client_id, user_id, access_token):
+    """Reset the stats of a term by deleting and re-creating it."""
+    found_sets = [user_set for user_set in get_user_sets(client_id, user_id)
+                  if user_set.set_id == set_id]
```

Comment:
Is it possible for multiple sets to share the same id? I thought it's supposed to be unique number.

Reply:
you're right, I'm overly protective here, just in case. It's written in a quite compact way and I don't expect these data structures to become huge, so I guess it's OK?

Reply:
Ah, I see, sure!